pysat/tests/test_files.py

```diff
@@ -136,6 +116,35 @@ def teardown(self):
             pass
         del self.testInst
 
+    def test_parse_delimited_filename(self):
```

Comment:
You have a few special words used by the method. While as written things should be ok, it could potentially cause some issues in the future. I'd recommend moving min to mins, dict to file_dict (?).

Reply:
fixed